test_regress/t/t_gantt_hier.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+# DESCRIPTION: Verilator: Verilog Test driver/expect definition
+#
+# Copyright 2024 by Wilson Snyder. This program is free software; you
+# can redistribute it and/or modify it under the terms of either the GNU
+# Lesser General Public License Version 3 or the Perl Artistic License
+# Version 2.0.
+# SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+
+# Test for bin/verilator_gantt,
+
+import vltest_bootstrap
+
+test.scenarios('vlt_all')
```

Comment:
```suggestion
test.scenarios('vlt')
```

Given this is testing a subcase of the larger gantt testing, I think we don't need to test vltmt, to speed up the general test suite.

Reply:
Could we instead test it only in `vltmt` mode? This will test the scenario when functions are traced from main thread and tasks from other threads and validate if timings are just.